deploy/roles/postgresql-server/templates/pg_hba.conf.j2

```diff
@@ -1,6 +1,6 @@
 {{ template_masthead }}
 # TYPE  DATABASE        USER            ADDRESS                 METHOD
 
-{% for setting in pp_hba_settings %}
+{% for setting in pg_hba_settings %}
```

Comment:
Since we've changed some variables from pp_XXX to pg_XXX, we should probably change all of them, and prefix the edb ones with edb_